spec/lib/webauthn/authentication_service_spec.rb

```diff
@@ -169,9 +183,9 @@
   end
 
   context "when the sha256 hash of the relaying party ID does not match the one in attestation.authData" do
-    let(:rp_id) { "bad_rp_id" }
-
     it "raises a InvalidRelyingPartyIdError" do
+      DiscourseWebauthn.stubs(:rp_id).returns("bad_rp_id")
```

Comment:
I think if we continue to pass the `rp_id` through to the service, we can avoid a stub here.